state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -626,14 +643,38 @@ pub mod pallet {
 		DepositBoosted {
 			deposit_address: TargetChainAccount<T, I>,
 			asset: TargetChainAsset<T, I>,
-			amount: TargetChainAmount<T, I>,
+			amounts: BTreeMap<BoostPoolTier, TargetChainAmount<T, I>>,
 			deposit_details: <T::TargetChain as Chain>::DepositDetails,
+			boost_id: BoostId,
```

Comment:
Also I just realised that boost_id is just the prewitnessed_deposit_id. It's a bit strange that we use these interchangeably despite them having different type aliases... Why do we distinguish these?